web/app_local.php

```diff
@@ -0,0 +1,73 @@
+<?php
```

Comment:
This environment shouldn't be included (it's being removed from other projects since it's not needed/broken).